cg/apps/madeline.py

```diff
@@ -1,65 +1,76 @@
-# -*- coding: utf-8 -*-
-import os.path
-import subprocess
+"""Code to handle madeline input and output"""
```

Comment:
Maybe we should try to be consistent defining all APIs to external tools as classes, as we have done for our own tools. Same could be done for madeline.

Reply:
I agree, will add that in the next PR